test/constants.ts

```diff
@@ -209,8 +209,11 @@ export const TEST_SERVICE_MAP_GRAPH = {
         label: 'order',
         size: 15,
         title: 'order\n\n Average duration: 90.1ms \n Error rate: 4.17% \n Request rate: 48',
-        borderWidth: 0,
-        color: 'rgba(158, 134, 192, 1)',
+        borderWidth: 3,
```

Comment:
question: are these changes being signed off from UX/product?

Reply:
Synched offline for sign off.